org.mixedrealitytoolkit.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -578,6 +579,9 @@ private GameObject FindInteractorGroupObject(XRBaseInteractor interactor)
                 }
             }
 
+            Assert.IsNotNull(interactorGroupObject, $"Interactor {interactor.name} ({interactor.GetType().Name}) is not managed by any interactor group. " + Environment.NewLine +
```

Comment:
Move this assert the "RegisterInteractor" call...closer to where the problem is.   Also to avoid a double warning caused by "UnregisterInteractor"

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/824/commits/ee35650ca1a6c7190044f23e5bf7270cb7900dd1 , resolving.